.github/workflows/deployDev.yml

```diff
@@ -94,3 +94,41 @@ jobs:
           azure_creds: ${{ secrets.AZURE_CREDENTIALS }}
           client_tarball: client.tgz
           deploy_env: ${{ inputs.deploy_env }}
+
+  smoke_test_front_and_back_end:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.deploy_env }}
+    needs: [deploy]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{env.NODE_VERSION}}
+      - name: Cache yarn
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Set up dependencies
+        working-directory: frontend
+        run: yarn install --prefer-offline
+      - name: Smoke test the env
+        uses: ./.github/actions/post-deploy-smoke-test
+        with:
+          base_domain_name: ${{ inputs.deploy_env }}.simplereport.gov
+
+  slack_alert:
+    runs-on: ubuntu-latest
+    if: ${{ always() && needs.smoke_test_front_and_back_end.result != 'success' }}
```

Comment:
This conditional is a little strange but apparently this `always()` is needed to fix the issue of this job getting skipped even if the right side of the conditional evaluates to true